streaming/text/convert/enwiki_text.py

```diff
@@ -67,29 +67,30 @@ def parse_args() -> Namespace:
 
 
 def process_split(in_root: str, out_root: str, compression: str, hashes: List[str],
-                  size_limit: int, progbar: int, leave: int, basenames: List[str],
+                  size_limit: int, progress_bar: int, leave: int, basenames: List[str],
                   split: str) -> None:
     """Process a dataset split.
 
     Args:
-        in_root (str): Input dataset path.
-        out_root (str): Output dataset path.
+        in_root (str): Local directory path of the input raw dataset.
+        out_root (str): Directory path to store the output MDS shard files.
         compression (str): Which compression to use, or empty string if none.
         hashes (List[str]): List of hashes to store of the shards.
         size_limit (int): Maximum shard size in bytes.
-        progbar (int): Whether to display a progress bar.
+        progress_bar (int): Whether to display a progress bar.
         leave (int): Whether to leave the progress bar.
         basenames (List[str]): List of input shard basenames.
         split (str): Split name.
     """
     split_dir = os.path.join(out_root, split)
     columns = {'text': 'str'}
-    with MDSWriter(local=split_dir,
+    with MDSWriter(out=split_dir,
                    columns=columns,
                    compression=compression,
                    hashes=hashes,
-                   size_limit=size_limit) as out:
-        if progbar:
+                   size_limit=size_limit,
+                   progress_bar=progress_bar) as out:
+        if progress_bar:
             basenames = tqdm(basenames, leave=leave)
```

Comment:
two progress bars again? have you run this?

Reply:
Yes, the two progress bar works in sync and I did ran ade20k example which has two progress bars.